bundle/regal/rules/testing/dubious_print_sprintf.rego

```diff
@@ -0,0 +1,20 @@
+# METADATA
+# description: dubious print sprintf
```

Comment:
Let's change this to "Dubious use of `print` and `sprintf`"

Reply:
Thanks for the comment. I will fix it.